r/PACKAGING.md

```diff
@@ -0,0 +1,128 @@
+
+Wait for the release candidate to be cut:
+
+- [ ] Release candidate!
+
+Make pull requests into the [autobrew](https://github.com/autobrew) and
+[rtools-packages](https://github.com/r-windows/rtools-packages) repositories
+used by the configure script on MacOS and Windows. These pull requests will
+use the release candidate as the source.
+
+- [ ] Pull request to modify 
+  [the apache-arrow autobrew formula]( https://github.com/autobrew/homebrew-core/blob/high-sierra/Formula/apache-arrow.rb) 
+  to update the release version, SHA256 checksum of the release source file, and any changes to dependencies and build steps that have changed in the
+  [copy of the formula we have of that formula in the Arrow repo](https://github.com/apache/arrow/blob/master/dev/tasks/homebrew-formulae/autobrew/apache-arrow.rb)
+- [ ] Pull request to modify
+  [the apache-arrow-static autobrew formula]( https://github.com/autobrew/homebrew-core/blob/master/Formula/apache-arrow-static.rb)
+  to update the version, SHA, and any changes to dependencies and build steps that have changed in the
+  [copy of the formula we have of that formula in the Arrow repo](https://github.com/apache/arrow/blob/master/dev/tasks/homebrew-formulae/autobrew/apache-arrow-static.rb)
+- [ ] Pull request to modify the 
+  [autobrew script](https://github.com/autobrew/scripts/blob/master/apache-arrow)
+  to include any additions made to
+  [r/tools/autobrew](https://github.com/apache/arrow/blob/master/r/tools/autobrew).
+- [ ] Pull request to modify the
+  [PKGBUILD script](https://github.com/r-windows/rtools-packages/blob/master/mingw-w64-arrow/PKGBUILD)
+  to reflect changes in
+  [ci/PKGBUILD](https://github.com/apache/arrow/blob/master/ci/scripts/PKGBUILD),
+  uncommenting the line that says "uncomment to test the rc".
+
+Prepare and check the .tar.gz that will be released to CRAN.
+
+- [ ] `git fetch upstream && git checkout release-X.X.X-rcXX && git clean -f -d`
+- [ ] Unset the `ARROW_HOME` environment variable and then run `make build` (copies Arrow C++ into tools/cpp, prunes some unnecessary components, and runs `R CMD build`)
+- [ ] `devtools::check_built("arrow_X.X.X.tar.gz")` locally
+- [ ] Run reverse dependency checks
+
+Wait for the official release...
+  
+- [ ] Release vote passed!
+- [ ] Create a CRAN-release branch from the official release commit
+- [ ] Pick any commits that were made to master since the release commit that
+  were needed to fix CRAN-related submission issues (e.g., fixed URLs, fixes
+  to )
+- [ ] Remove badges from README.md
+- [ ] Run `urlchecker::url_check()` on the R directory
+- [ ] Bump the Version in DESCRIPTION to X.X.X
+- [ ] Create a PR entitled `WIP: [R] Verify CRAN release-10.0.1-rc0`. Add
+  a comment `@github-actions crossbow submit --group r` to run all R crossbow
+  jobs against the CRAN-specific release branch.
+- [ ] Regenerate arrow_X.X.X.tar.gz (i.e., `make build`)
+
+Update autobrew and r-windows PRs to use the *release* instead of the
```

Comment:
I would note that lately Jeroen has been merging these PRs before the vote closes so that we can test. So this may require new PRs. 

Also you can make all of the version bumps in the autobrew formulae in the initial PR now, no need to follow up here. It uses the RC URL as "mirror", so before the vote passes, the release URL will 404 and it falls back to the RC. After, it uses the official release. So it's only the rtools-packages one that needs switching.